### PR TITLE
build(client): Add biome formatting scripts

### PR DESCRIPTION
**Pull request by tylerbutler**
## Summary

Adds several scripts that can be run to convert projects in the client release group to use the biome formatter. The process can be effectively reversed by running another script, so projects can switch between prettier and biome using the scripts and the build will continue to work.

After enabling biome formatting, you can manually format using biome with the `format:biome`script.

Once projects are completely on biome and we no longer need to port changes between prettier branches and biome branches, then we can remove prettier completely using the remove-prettier script. We should only do that once the last release branch using prettier is no longer in service.

### add-biome.sh

This script adds biome formatting scripts to a project. It also adds a local biome config file to the project if needed.

The `format:prettier` and `check:prettier` tasks are left intact, and the `format` task still calls prettier. This
enables one to manually format using biome while leaving the overall build using prettier.

After running this script, you can clean up the package.json files by running `pnpm policy-check:fix`.

**This PR includes the changes made by running this script across the client release group.**

### enable-biome.sh

This script enables biome formatting in a project. It updates the `format` and `check:format` scripts to call biome. It
should only be run on projects that have already had biome added to it.

### enable-prettier.sh

This script enables prettier formatting in a project. It updates the `format` and `check:format` scripts to call
prettier. It's the opposite of the enable-biome.sh script.

### remove-prettier.sh

Once a project is completely switched to biome and no longer needs prettier, this script will remove the prettier
dependency and any related scripts and config files.

### CODE REVIEW

**Comment by alexvy86** [comment]
I think something's not right here 😄 
image

File: packages/dds/map/src/test/mocha/directoryFuzzTests.spec.ts [hunk 1]
```diff
@@ -370,8 +369,7 @@ describe("SharedDirectory fuzz Create/Delete concentrated", () => {
 			defaultTestCount: 200,
 			// The seeds below fail only when rebaseProbability is non-zero ADO:6044
 			skip: [
-				13, 40, 43, 55, 66, 93, 94, 107, 110, 123, 136, 148, 160, 163, 168, 172, 177, 191,
-				196,
```
Comment (ChumpChief) [inline packages/dds/map/src/test/mocha/directoryFuzzTests.spec.ts:374]: Much better :-P

File: packages/dds/map/package.json [hunk 2]
```diff
@@ -71,15 +71,14 @@
 		"build:test:esm": "tsc --project ./src/test/tsconfig.json",
 		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
 		"check:biome": "biome check .",
-		"check:prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"check:format": "fluid-build --task check:format .",
```
Comment (ChumpChief) [inline packages/dds/map/package.json:74]: What's the difference between check:biome/check:format (and format/format:biome)?  Is one of the pair "correct" to call?  Do we need both?

I see you talk about these in the PR description but it doesn't quite seem to match up with what I see here?

**Review by ChumpChief — APPROVED** [review]
Seems mostly fine to me (love that this will speed up the build!), though I'm a little confused about the scripts.

**Pull request merged**